ui/webui/resources/BUILD.gn

```diff
@@ -30,18 +29,18 @@ group("grdp") {
     ":leo_web_components",
     ":static_grdp",
   ]
-  if (include_polymer) {
+  if (!is_android && !is_ios) {
```

Comment:
Why do we need the os buildflags? I think we only had `include_polymer` because otherwise it broke on android/ios? We shouldn't be using os buildflags like this

Reply:
Maybe, but this mirrors precisely what upstream is doing:

https://chromium.googlesource.com/chromium/src/+/27af2840dc73c5e5a0cb13e368b2a44820ba7431

```
-assert(include_polymer)
+assert(!is_android && !is_ios)
```

I think the use of `include_polymer` precisely to exclude configs that could build polymer, and that's how it has been.

Reply:
huh, that's very odd, but I guess if it matches upstream then it's ok